rllib/env/utils.py

```diff
@@ -47,6 +47,16 @@ def _gym_env_creator(env_context: EnvContext, env_descriptor: str) -> gym.Env:
     # Try creating a gym env. If this fails we can output a
     # decent error message.
     try:
-        return gym.make(env_descriptor, **env_context)
+        # Special case: Atari not supported by gymnasium yet -> Need to use their
+        # GymV26 compatibility wrapper class.
+        # TODO(sven): Remove this if-block once gymnasium fully supports Atari envs.
+        if env_descriptor.startswith("ALE/"):
+            return gym.make(
```

Comment:
is this the wrapper class?

Reply:
Yes, it's a special wrapper env ID that you can use to wrap an Atari env with the old gym package and make it work with gymnasium. It requires the actual Atari ID (but with "ALE/" in front :) ). For now, I figured it would be nice to have RLlib automatically do the right thing.